README.md

```diff
@@ -57,6 +57,26 @@ It involves mainly the addition of:
 -   the murano agent configuration file: It includes different murano-agent configuration files for services to be included in the different
   linux distributions.
 
+#### Installing FIWARE Murano requirements by an script
+fiware-murano repository contains a script which installs all the FIWARE specific requirements. This script is in charge of:
+- creating the virtualenv
+- installing python dependences,
+- generating the core library zip file
+- uploading into murano instance
+
```

Comment:
- creating the virtualenv.
- installing python dependences.
- generating the core library zip file.
- uploading into murano instance.

Nevertheless, appart from the creation of the virtualenv, the oder do not have a clear step what the user need to do, I suggest to put the concrete command to execute them, specially in step 2 or 3 where it is the first time, I suppose, that we talk about core library and murano instance.